### DISCUSSION TITLE
Ability to click on "previous contacts" panel to right of Live QSO fields

### DISCUSSION THREAD

**Discussion by ofadam** [magicbug/Cloudlog#1338] [Ideas / Feature Requests]
When hunting Parks on the Air contacts especially, it might be helpful to be able to click on the QSOs listed in the "previous contacts" panel on the Live QSO page to then trigger the QSO lightbox for that prior contact. It would still keep anything entered for that live QSO intact, but would allow you to see more details from previous QSOs with that particular contact.

**Comment by AndreasK79** [comment]
@ofadam I thought it did this, but only when it first loads, not when loading after looking up previous records. I agree that this should be there. I'll take a look at it.

**Comment by AndreasK79** [comment]
@ofadam I've added it to the codebase now.

**Comment by ofadam** [comment]
Just tested and it works great, thanks so much!
